spec/mongoid/contextual/memory_spec.rb

```diff
@@ -944,7 +948,7 @@
       end
 
       it "returns the values" do
-        expect(plucked).to eq([ "hobrecht", "friedel" ])
+        expect(plucked).to eq([ "hobrecht", "friedel", nil ])
```

Comment:
Please separate the case of nil values to their own tests for clarity under a `context "when value in database is nil"` or similar.

Reply:
Done.